support-frontend/cloud-formation/cfn.yaml

```diff
@@ -50,8 +52,7 @@ Resources:
   AutoScalingGroup:
     Type: AWS::AutoScaling::AutoScalingGroup
     Properties:
-      AvailabilityZones: !GetAZs
-      VPCZoneIdentifier: !Ref Subnets
+      VPCZoneIdentifier: !Ref PrivateVpcSubnets
```

Comment:
Do we not need to set availability zones any more?

Reply:
Yep good point. `AvailabilityZones` isn't needed because we supply `VPCZoneIdentifier` (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-availabilityzones). So as long as we supply a subnet for each AZ, it has the same effect. At one point while experimenting with moving subnets around, having it there caused issues because it got out of sync with `VPCZoneIdentifier` (`!GetAZs` always returns all the AZs for the region). So I decided to remove it across the board to avoid making the migration process harder.

Maybe there's an argument that having it there enforces a subnet in each AZ. But I feel like since the subnet parameters will be the same across every single app the chance of error is quite low.